docker/ubuntu20.04-sydr-fuzz/Dockerfile

```diff
@@ -33,4 +33,24 @@ RUN git clone https://github.com/llvm/llvm-project.git && cd llvm-project && \
 RUN echo /usr/lib/x86_64-unknown-linux-gnu > /etc/ld.so.conf.d/libc++.conf && \
     ldconfig
 
+RUN apt-get -y install lcov cargo pkg-config libglib2.0-dev \
+    gcc-10 g++-10 gcc-10-plugin-dev gcc-10-multilib
+
+RUN cd / && git clone https://github.com/AFLplusplus/AFLplusplus.git && cd AFLplusplus && \
```

Comment:
You are already in /

Reply:
You may clone afl++ in the same RUN command that builds it. Do not forget to remove afl++ sources directory later.

Reply:
Done